src/components/NavBar.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+
+export const NavBar = () => {
+  const location = useLocation();
+  const [isActive, setIsActive] = useState<string>('');
+
+  useEffect(() => {
+    const currentLink = location.pathname;
+
+    if (currentLink === '/' || currentLink === '/home') {
+      setIsActive('home');
+    } else if (currentLink.startsWith('/people')) {
+      setIsActive('people');
+    } else {
+      setIsActive('');
+    }
+  }, [location]);
+
+  const handleClick = (link: string) => {
+    setIsActive(link);
+  };
```

Comment:
it is a creative way to check if link is active but in NavLink you can check if it's active ;) https://reactrouter.com/en/main/components/nav-link
Try to use this built-in feature instead of creating state isActive